couch/tests/compose/compose_v2.yaml

```diff
@@ -1,8 +1,48 @@
-version: '3.5'
+version: "3.5"
+
+networks:
+  network:
 
 services:
-  couch:
-    image: datadog/docker-library:couch_${COUCH_VERSION}
```

Comment:
@therve Is this image only used here?

Shall we it ? https://github.com/DataDog/docker-library/blob/master/couch/2.3/Dockerfile